### ISSUE TITLE
What is the model used in this notebook?

### ISSUE THREAD

**Issue by SkeletonOne** [Lexie88rus/Activation-functions-examples-pytorch#1]
Hi, thanks for your excellent work. I'd like to know about the model used in your work. Is it a 3-branch residual network in shake-shake?

Besides, what is the create_viz used for? Sincerely looking forward for your response.

**Comment by Lexie88rus** [comment]
Hi @SkeletonOne ! Thank you so much for your comment! I am so sorry for the late response.

I used shake-shake network from [this repository](https://github.com/dnddnjs/pytorch-cifar10).

create_viz is for the visualization of the loss function landscapes. I used the code from [this article](https://towardsdatascience.com/visualizing-loss-landscape-of-deep-neural-networks-but-can-we-trust-them-3d3ae0cff46e). Find the original code [in this repo](https://github.com/tomgoldstein/loss-landscape).

**Comment by SkeletonOne** [comment]
Thanks! I got it.
